Commands/MasteryCommands.cs

```diff
@@ -46,6 +46,20 @@ public static void GetMastery(ChatCommandContext ctx, string weaponType = "")
             MasteryData data = wd[type];
             ctx.Reply(GetMasteryDataStringForType(type, data));
         }
+        
+        [Command("details", "d", "", "Displays detailed information about benefits from mastery for your equipped weapon")]
```

Comment:
This can be OK for providing more specific information solely on the weapon buffs, but you can view this same information (along with the HP buff from XP) by using `.playerinfo`. I'd probably lean towards providing a consolidated stat display command there so that as this (and the other buff systems) come online that the user can see all their buffs in one place rather than needing to type 1 command for each system.